pngcanvas.py

```diff
@@ -47,6 +47,16 @@ def gradient_list(start, end, steps):
             for i in range(steps + 1)]
 
 
+def rgb2rgba(rgb):
+    """Take a row of RGB bytes, and convert to a row of RGBA bytes."""
+    rgba = []
```

Comment:
This might be better/faster actually:

    rgba = ((rgb[i], rgb[i+1], rgb[i+2], 255) for i in xrange(0, len(rgb), 3))

NOTE: May need to replace `xrange` with `range` for Py3 compatibility.